components/common-data-items.js

```diff
@@ -151,6 +151,21 @@ export function SampleDataItems({
           </DataItemValue>
         </>
       )}
+      {truthyOrZero(item.sorted_fraction) && (
```

Comment:
Just do `{item.starting_amount && (`. `truthyOrZero` is purely for numeric values that have a displayable zero value (the comment for the function describes what purpose it serves). `item.starting_amount` appears to be an embedded object, not a numeric value.

Reply:
done